.travis.yml

```diff
@@ -10,11 +10,12 @@ python:
   - '3.5'
   - '3.6'
   - '3.7-dev'
+node_js: "8.11"
 
```

Comment:
I think we should use lts/* node as is done in (some) of our other projects?

Reply:
What do you mean? 8.11 is the latest LTS Node currently (check https://nodejs.org/en/).

Reply:
Ah... checked the docs of Travis and now I understand what you mean. Will update the PR...